climbdex/api.py

```diff
@@ -154,3 +154,46 @@ def api_save_ascent(
     except Exception as e:
         logging.error(f"Error in save_ascent: {str(e)}", exc_info=True)
         return flask.jsonify({"error": str(e)}), 500
+
+@blueprint.route("/api/v2/climbs", methods=["POST"])
```

Comment:
```suggestion
@blueprint.route("/api/v1/climbs", methods=["POST"])
```

Should be v1, a v2 would be if we made breaking changes to a v1 version of the endpoint

Reply:
I guess my assumption was that it would be the same version as the boardlib api? But it's different so I kinda get it.

Reply:
Ah I see - that API is actually the Aurora API. Maybe it would have been better to do a complete copy of the Aurora API but we're pretty far from that currently, so I'm thinking of the Climbdex API as something totally separate that sometimes corresponds to similar Aurora API calls.